recipes/libraw/all/conanfile.py

```diff
@@ -66,6 +67,8 @@ def source(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
+        if Version(self.version) <= "0.19.5":
+            tc.variables["CMAKE_CXX_STANDARD"] = 11
```

Comment:
in Conan 2.0 I think this effectively nullifies all the effects of the `compiler.cppstd` setting. 

If this recipe is not compatible with C++17 or higher, I believe we should be using: https://docs.conan.io/2/reference/tools/build.html#conan-tools-build-check-max-cppstd

Setting it to C++11 here when the package ID has c++17 (or higher) can be problematic if the `register` keyword is in a public header.

Reply:
@jcar87 
Thank your for your review!
I updated this PR to use check_max_cppstd.

Please review it again.

Reply:
fixed